.github/workflows/main.yml

```diff
@@ -169,6 +169,17 @@ jobs:
           path: test_artifacts/
           retention-days: 5
 
+  build_on_rust_1_66:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install Rust 1.66
+      run: rustup install 1.66
+
+    - name: Build on rust 1.66
+      run: cd rust && cargo +1.66 build --ignore-rust-version
```

Comment:
Can we drop the option `--ignore-rust-version`, IMHO, if the rust compiler version is older than the required rust version, we should not proceed to build the target. 

```
       --ignore-rust-version
           Build the target even if the selected Rust compiler is older than
           the required Rust version as configured in the project’s
           rust-version field.
```

Reply:
The `ignore-rust-version` is for other crates.